File: second/java/Person.java
public class Person {
    private int identification;
    private String names;
    private String lastNames;
    private float weight;
    private float stature;
    private int selection;

    public Person(int identification, String names, String lastNames, float weight, float stature, int selection) {
        this.identification = identification;
        this.names = names;
        this.lastNames = lastNames;
        this.weight = weight;
        this.stature = stature;
        this.selection = selection;
    }

    public int getIdentification() {
        return this.identification;
    }

    public String getNames() {
        return this.names;
    }

    public String getLastNames() {
        return this.lastNames;
    }

    public float getWeight() {
        return this.weight;
    }

    public float getStature() {
        return this.stature;
    }

    public int getSelection() {
        return this.selection;
    }
}

File: second/php/libreria/person.php
<?php
class Person{
    private int $id;
    private string $names;
    private string $lastNames;
    private int $weight;
    private int $stature;
    private int $selection;

    public function __construct(int $id, string $names, string $lastNames, int $weight, int $stature, int $selection) {
        $this->id = $id;
        $this->names = $names;
        $this->lastNames = $lastNames;
        $this->weight = $weight;
        $this->stature = $stature;
        $this->selection = $selection;
    }
    public function getId(): int{
        return $this->id;
    }
    public function getNames(): string{
        return $this->names;
    }
    public function getLastNames() : string {
        return $this->lastNames;
    }
    public function getWeight(): int{
        return $this->weight;
    }
    public function getStature(): int{
        return $this->stature;
    }
    public function getSelection(): int{
        return $this->selection;
    }
}
?>